clients/java/src/main/java/io/camunda/zeebe/client/api/command/FinalCommandStep.java

```diff
@@ -15,32 +15,37 @@
  */
 package io.camunda.zeebe.client.api.command;
```

Comment:
❓ At some point, we need to move this class to the new package name as well. Better we do that now, I guess? Can this be considered a breaking change? 🙈

Reply:
This is going to be a breaking change, so it is not doable now

Reply:
So, we're introducing the breaking change with `8.8` then?

Reply:
This is the plan I think